tasks/styles.js

```diff
@@ -54,6 +54,8 @@ module.exports = (gulp, $, pkg) => {
       .pipe($.if(options.production, $.replace(copyrightPlaceholder, copyrightNotice)))
       .pipe($.if(options.production, $.cleanCss(cleanCssOptions)))
       .pipe($.if(options.production, $.rename({ suffix: '.min' })))
+      .pipe(gulp.dest(file => file.base))
```

Comment:
> * Styles are now also compiled in place

This causes compiled files to be written in the **assets** folder containing uncompiled source files:
```
$ git status
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	assets/scss/fractal.css
	assets/scss/vendor/photoswipe/default-skin/default-skin.css
	assets/scss/vendor/photoswipe/main.css
```
That cannot have been the intention, so I will commit a quick follow-up fix to remove this line.

Reply:
https://github.com/netzstrategen/gulp-task-collection/pull/22 fixes this problem.